modules/librewolf/hm.nix

```diff
@@ -0,0 +1,29 @@
+{ config, lib, ... }:
```

Comment:
> A copy of the Firefox module

For reference:

```console
$ diff modules/firefox/hm.nix  modules/librewolf/hm.nix
14c14
<   options.stylix.targets.firefox = {
---
>   options.stylix.targets.librewolf = {
16c16
<       config.lib.stylix.mkEnableTarget "Firefox" true;
---
>       config.lib.stylix.mkEnableTarget "Librewolf" true;
19c19
<       description = "The Firefox profile names to apply styling on.";
---
>       description = "The Librewolf profile names to apply styling on.";
25,27c25,27
<   config = lib.mkIf (config.stylix.enable && config.stylix.targets.firefox.enable) {
<     programs.firefox.profiles = lib.listToAttrs
<       (map makeProfileSettingsPair config.stylix.targets.firefox.profileNames);
---
>   config = lib.mkIf (config.stylix.enable && config.stylix.targets.librewolf.enable) {
>     programs.librewolf.profiles = lib.listToAttrs
>       (map makeProfileSettingsPair config.stylix.targets.librewolf.profileNames);
```

> don't think there's a better way to do it.

Consider adding a comment for future maintenance:

```suggestion
# This module is a copy of the Firefox module.

{ config, lib, ... }:
```

Reply:
I added a similar comment to the Firefox module also.